addons/account/models/account_invoice.py

```diff
@@ -1410,7 +1410,14 @@ def _compute_base_amount(self):
     currency_id = fields.Many2one('res.currency', related='invoice_id.currency_id', store=True, readonly=True)
     base = fields.Monetary(string='Base', compute='_compute_base_amount')
 
-
+    # DO NOT FORWARD-PORT!!! ONLY FOR v10
```

Comment:
Just curious... Is it because odoo 11 uses Python 3 and it does not have these kind of problems anymore? Or is there any new Odoo feature I'm missing?

Thanks!

Reply:
@Yajo https://github.com/odoo/odoo/pull/23563 ;)
In v11, `convert_to_column` accepts an extra `values` dict that can be used to force the rounding. Thing is, this values MUST be in the dict, the currency won't be fetched automatically.
https://github.com/odoo/odoo/blob/11.0/odoo/fields.py#L1265-L1268